src/lib/components/collections/collections-list/ImportCollection.svelte

```diff
@@ -220,113 +216,126 @@
+        class="format-btn"
         on:click={() => {
-          handleImport();
-        }}
-      >
-        <span class="me-3">
-          {#if isLoading}
-            <Spinner size={"16px"} />
-          {/if}</span
-        > Import Collection</button
-      >
-
-      <p class="empty-data-error">
-        {#if isDataEmpty && !importData}
-          Please Paste or Upload your file in order to import the workspace
-        {/if}
-      </p>
-      <p
-        class="importData-whiteColor mb-2"
-        style="font-size: 14px;font-weight:300"
+          onClick(false);
+        }}>Close</button
       >
-        OR
-      </p>
       <button
-        class="btn-primary border-0 w-100 py-2 fs-6 rounded"
+        class="format-btn collection-btn"
         on:click={() => {
-          onClick(false);
-          handleCreateCollection();
-        }}>Create Collection</button
+          handleError();
+        }}>+ Collection</button
       >
     </div>
   </div>
+</ModalWrapperV1>
+{#if isResponseLoading}
+  <FetchDataProgressBar {onClick}></FetchDataProgressBar>
 {/if}
 
-<style>
+<ModalWrapperV1
+  title={"New Collection"}
+  type={"dark"}
+  width={540}
+  zIndex={9}
+  isOpen={!isLoading && !isSyntaxError}
+  handleModalState={onClick}
+>
+  <div style="font-size: 14px;" class="importData-lightGray">
+    <p>Paste your YAML/JSON file</p>
+  </div>
+  <div class="textarea-div rounded border-0">
+    <textarea
+      on:input={(e) => handleInputField(e)}
+      bind:value={importData}
+      class="form-control border-0 rounded bg-blackColor"
+    />
+  </div>
+  <div style="font-size: 14px;" class="importData-lightGray">
+    <p class="mb-1">Drag and drop your YAML/JSON file</p>
+  </div>
+  <div>
+    <FileInput
+      value={uploadCollection.file.value}
+      maxFileSize={100}
+      onChange={handleLogoInputChange}
+      resetValue={handleLogoReset}
+      editValue={handleLogoEdit}
+      labelText=""
+      labelDescription="Drag and drop your YAML/JSON file"
+      inputId="upload-collection-file-input"
+      inputPlaceholder="Drag and Drop or"
+      isRequired={false}
+      supportedFileTypes={[".yaml", ".json"]}
+      showFileSizeError={uploadCollection.file.showFileSizeError}
+      showFileTypeError={uploadCollection.file.showFileTypeError}
+      type={"file"}
+      fileTypeError="This file type is not supported. Please reupload in any of the following file formats."
+      fileSizeError="The size of the file you are trying to upload is more than 100 KB."
+    />
+  </div>
+
+  <div
+    class="d-flex flex-column align-items-center justify-content-end rounded mt-4"
+  >
+    <button
+      class="btn-primary d-flex align-items-center justify-content-center border-0 w-100 py-2 fs-6 rounded"
+      on:click={() => {
+        handleImport();
+      }}
+    >
+      <span class="me-3">
+        {#if isLoading}
+          <Spinner size={"16px"} />

```

Comment:
Lets remove px from here as well

Reply:
Can we create a separate ticket for Spinner as we did for other components like modal and dropdown